privacyidea/lib/token.py

```diff
@@ -862,13 +862,11 @@ def import_token(serial, token_dict, default_hashlib=None, tokenrealms=None):
     :return: the token object
     """
     init_param = {'serial': serial,
-                  'type': token_dict['type'],
                   'description': token_dict.get("description",
-                                                "imported"),
-                  'otpkey': token_dict['otpkey'],
-                  'otplen': token_dict.get('otplen'),
-                  'timeStep': token_dict.get('timeStep'),
-                  'hashlib': token_dict.get('hashlib')}
+                                                "imported")}
+    for p in ['type', 'otpkey', 'otplen', 'timeStep', 'hashlib', 'tans']:
+        if p in token_dict:
+            init_param[p] = token_dict[p]
```

Comment:
This slightly changes behavior: Currently, if ``token_dict["hashlib"]`` is None, ``init_param["hashlib"]`` is None. With this PR, ``init_param`` does not contain a key ``hashlib``.
But I think this won't cause problems ...?

Apart from that, I think having the token-specific parameter names in ``import_token`` is not perfect in terms of modularity. But I can imagine changing this would require some major refactoring, so I would keep it as it is for now.